src/script.ts

```diff
@@ -655,3 +655,10 @@ function scrollToCurrentGame(): void {
     lastGame.scrollIntoView();
   }
 }
+function changeOpeningMoveColor() {
+  const color = "#c5c5c5";
```

Comment:
the color is a placeholder and i will change that later

Reply:
lgtm
Later will make it optional